gdk4/src/toplevel_size.rs

```diff
@@ -7,9 +7,11 @@ use glib::translate::*;
 glib::wrapper! {
 
 #[doc(alias = "GdkToplevelSize")]
-    pub struct ToplevelSize(BoxedInline<ffi::GdkToplevelSize>);
+    pub struct ToplevelSize(Boxed<ffi::GdkToplevelSize>);
 
     match fn {
+        copy => |ptr| glib::gobject_ffi::g_boxed_copy(ffi::gdk_toplevel_size_get_type(), ptr as *mut _) as *mut ffi::GdkToplevelSize,
+        free => |ptr| glib::gobject_ffi::g_boxed_free(ffi::gdk_toplevel_size_get_type(), ptr as *mut _),
         type_ => || ffi::gdk_toplevel_size_get_type(),
```

Comment:
A pointer that can't be copied/freed needs manual bindings. It's not anything that can be represented with the gobject-introspection types and actual behaviour / semantics of such types need to be decided on a case by case basis.

This one here is a boxed type though so that should be fine? Boxed types can always be copied/freed.

Reply:
But it's not a boxed type, it's just a registered pointer inside GTK... I think you need something manual.

Reply:
Oh, then yes this is wrong and needs to be done manually. Thanks for catching that :)

Reply:
Thanks @ocrete